astroquery/splatalogue/load_species_table.py

```diff
@@ -36,10 +38,42 @@ def find(self, s, flags=0, return_dict=True,):
             return out.values()
 
 
-def species_lookuptable(filename='species.json'):
-    with open(data_path(filename), 'r') as f:
-        J = json.load(f)
+def species_lookuptable(filename='splat-species.json', recache=False):
+    """
+    Function to format the species ID results from scraping Splatalogue
+    into a ``SpeciesLookuptable`` object.
+
+    The first step is to check whether or not a cached result exists;
+    if not, we run the scraping routine and use this result. Otherwise,
+    load and use the cached result.
+
+    The ``recache`` flag can be used to force a refresh of the local
+    cache.
+
+    Parameters
+    ----------
+    filename : str, optional
+        Name of the file cache, by default 'splat-species.json'
+    recache : bool, optional
+        If True, force refreshing of the JSON cache, by default False
 
+    Returns
+    -------
+    ``lookuptable``
+        ``SpeciesLookuptable`` object
+    """
+    file_cache = data_path(filename)
+    # check to see if the file exists; if not, we run the
+    # scraping routine
+    if os.path.isfile(file_cache) is False:
+        J = get_json_species_ids(filename)
+    else:
+        with open(data_path(filename), 'r') as f:
+            J = json.load(f)
+    # add an option to refresh the cache and use the latest
+    # result
+    if recache:
```

Comment:
why not just add this as `or recache` on [line 68 ](https://github.com/astropy/astroquery/pull/1772/files#diff-24931fe7da5bc09bd7bf9f47b9bfd0ddR68) above?

Reply:
You're right, I'll make that change :)